<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Redactor</class>
 <widget class="QDialog" name="Redactor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Корректировка</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/bg_image/images/Фон_1.PNG);</string>
  </property>
  <widget class="QLineEdit" name="Otvet_1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/bg_image/images/White);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="Otvet_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>430</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/bg_image/images/White);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="Otvet_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/bg_image/images/White);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="Vopros">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string> Введите 1-ый вопрос.</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="Otvet1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Введите 1-ый ответ.</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="Otvet2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>Введите 2-ый ответ.</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="Otvet3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>400</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Введите 3-ый ответ.</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="Otvet4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>500</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Введите 4-ый ответ.</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="Otvet_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>530</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-image: url(:/bg_image/images/White);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="Gotovo">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>540</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/bg_image/images/White);</string>
   </property>
   <property name="text">
    <string>Готово</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Vopros_1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>681</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/bg_image/images/White);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_1">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>220</y>
     <width>31</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>320</y>
     <width>31</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>420</y>
     <width>31</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_4">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>520</y>
     <width>20</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/bg_image/images/White);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
  </widget>
  <zorder>rb_1</zorder>
  <zorder>Otvet_1</zorder>
  <zorder>Otvet_3</zorder>
  <zorder>Otvet_2</zorder>
  <zorder>Vopros</zorder>
  <zorder>Otvet1</zorder>
  <zorder>Otvet2</zorder>
  <zorder>Otvet3</zorder>
  <zorder>Otvet4</zorder>
  <zorder>Otvet_4</zorder>
  <zorder>Gotovo</zorder>
  <zorder>Vopros_1</zorder>
  <zorder>rb_2</zorder>
  <zorder>rb_3</zorder>
  <zorder>rb_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
